pipelines/bazel-release.yml

```diff
@@ -215,6 +215,8 @@ steps:
     agents:
       - queue=windows
     command: |
+      set BUILDKITE_MESSAGE=%BUILDKITE_MESSAGE:~0,1000%
```

Comment:
why do we need this and the bazelci fix?

Reply:
Because the release pipeline is not using the bazelci.py.

Reply:
And the bazelci fix is exactly as you recommended?